plugins/slo/pkg/apis/slo/slo.proto

```diff
@@ -121,65 +107,36 @@ message Alert {
 }
 
 message ServiceLevelObjective {
-  string id = 1;
-  string name = 2;
-  string datasource = 3;
-  string description = 4;
-  repeated Service services = 5;
-  string monitorWindow = 6; // 7d, 28d or 30d
-  string metricDescription = 8;
-  string budgetingInterval = 9; // in the format 5m, 1h etc..
-  repeated Label labels = 10;
-  repeated Target targets = 11;
-  repeated Alert alerts = 12;
-}
-
-message Target {
-  uint64 ValueX100 = 1;
-  google.protobuf.Duration timeWindow = 2; // FIXME: should be changed to str
-}
-
-message ServiceLevelObjectiveList {
-  repeated ServiceLevelObjective items = 1;
-}
-
-// TODO : rework this
-message SLOStatus {
-  core.Reference slo = 1;
-  int64 timestamp = 2;
-  Evaluation status = 3;
-  Evaluation budget = 4;
+  string name = 1;
+  string datasource = 2;
+  string monitorWindow = 3; // 7d, 28d or 30d
+  string budgetingInterval = 4; // in the format 5m, 1h etc..
```

Comment:
Use google.protobuf.Duration for the these fields, so that it will turn the field into native time structs

Reply:
Done, except the monitorWindow which is intended to be an enum.